vllm_ascend/platform.py

```diff
@@ -31,15 +31,12 @@
     # register custom ops into torch_library here
     import vllm_ascend.vllm_ascend_C  # type: ignore  # noqa: F401
 
-except ImportError as e:
-    if not str(
-            e
-    ) == "dynamic module does not define module export function (PyInit_vllm_ascend_C)":
-        logging.warning(
-            "Warning: Failed to register custom ops, all custom ops will be disabled"
-        )
-    else:
-        CUSTOM_OP_ENABLED = True
+except ImportError:
+    logging.warning(
```

Comment:
use logger from vllm is better. And line 39 can be moved to L33. We can fix the nit later.